docs/Neogeo-CD.md

```diff
@@ -0,0 +1,46 @@
+***
+_ The Neo Geo CD is the second home video game console of SNK Corporation's Neo Geo family, released on September 9, 1994, four years after its cartridge-based equivalent. The system was originally priced at US$399, or £399. It was discontinued in 1997 with sales of 570,000 systems._
+
+***
+
+| Emulator | Rom Folder | Extension | BIOS |  Controller Config |
+| :---: | :---: | :---: | :---: | :---: |
+| [lr-neocd](https://github.com/libretro/neocd_libretro) | neogeo  | .chd .cue .zip | uni-bioscd.rom | /opt/retropie/configs/neogeo/retroarch.cfg |
```

Comment:
Add `fbneo`. Remove `.zip` as supported file.

Reply:
I used a zip file on my own unmodified system and it worked fine.